settings/DevHome.Settings/DevHome.Settings.csproj

```diff
@@ -17,19 +17,29 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
+    <None Remove="NativeMethods.txt" />
     <None Remove="Views\AboutPage.xaml" />
     <None Remove="Views\AccountsPage.xaml" />
     <None Remove="Views\ExtensionsPage.xaml" />
+    <None Remove="Views\FeedbackPage.xaml" />
     <None Remove="Views\LoginUIDialog.xaml" />
     <None Remove="Views\PreferencesPage.xaml" />
   </ItemGroup>
+  <ItemGroup>
+    <AdditionalFiles Include="NativeMethods.txt" />
+  </ItemGroup>

```

Comment:
Are the NativeMethods.txt changes here needed?

Reply:
No, removed.